qemu/tests/win_heavyload.py

```diff
@@ -139,9 +139,16 @@ def add_option(cmd, key, val):
             if not installed and download_url:
                 utils.system("rm -f %s/HeavyLoad*.exe" % tmp_dir)
                 session.cmd("del /f /s %sHeavyLoad*.exe" % dst)
-            if loop_session_cmd(session, check_running_cmd):
-                if not loop_session_cmd(session, stop_cmd):
-                    raise error.TestFail("Unable to terminate heavyload "
-                                         "process")
+            for i in range(3):
```

Comment:
1. you can use loop_session_cmd directly
2. I think no need to repeat so many times, as it just a clean step, it won't effect the test